scripts/Deploy-OxaStamp.ps1

```diff
@@ -133,7 +136,16 @@ Param(
 
         [Parameter(Mandatory=$false)][string]$DeploymentVersionId="",
 
-        [Parameter(Mandatory=$false)][switch]$EnableMobileRestApi=$false
+        [Parameter(Mandatory=$false)][switch]$EnableMobileRestApi=$false,
+        
+        [Parameter(Mandatory=$true)][string]$BranchName = "oxa/devfic",
+
+        [Parameter(Mandatory=$false)][string]$DeploymentType="upgrade",
+        
+        [Parameter(Mandatory=$false)][string]$Slot="slot1",
+
+        [Parameter(Mandatory=$true)][ValidateSet("Prod", "Int", "BVT", "")][string]$Cloud="BVT"
```

Comment:
you may want to go with lower case options

Reply:
done